### PR TITLE
ISSUE-238: search_api warnings for missing view mode configuration triggered on index

### LINKED ISSUE #238: search_api warnings for missing view mode configuration triggered on index
# What?
When indexing 2 warnings are triggered by Drupal:

```
The field Rendered HTML output (rendered_item) on index Drupal Content to Solr 8 is missing view mode configuration for some datasources or bundles. Please review (and re-save) the field settings.
```
and
```
The field IIIF Image Tag for ADO (rendered_item_iiif_img) on index Drupal Content to Solr 8 is missing view mode configuration for some datasources or bundles. Please review (and re-save) the field settings.
```

Just need to add some configs for the`strawberryfield_flavor_datasource` bundle so the warning doesn't trigger. PR to come shortly.

### PR DESCRIPTION
**Pull request by aksm**
Resolves #238.

### CODE REVIEW

File: config/sync/search_api.index.default_solr_index.yml [hunk 1]
```diff
@@ -356,8 +357,9 @@ field_settings:
           page: ''
           webform: ''
         strawberryfield_flavor_datasource:
-          digital_object: search_index
-          digital_object_collection: search_index
+          digital_object: ''
```
Comment (DiegoPino) [inline config/sync/search_api.index.default_solr_index.yml]: @aksm cool. So... why still digital_object and the other bundles, are returning those values?

Comment (aksm) [reply config/sync/search_api.index.default_solr_index.yml]: @DiegoPino Thinking about it now a little more carefully I'm not entirely sure. I was just copying the empty values from rendered_item above. Should they just be removed entirely?

**Comment by DiegoPino** [comment]
I think the answer is checking on the debugger and see what the datasource method (the one that requests bundles) is actually returning

**Comment by DiegoPino** [comment]
@aksm just to be sure (I don't see a commit message) is this what the debugger shows?

**Comment by aksm** [comment]
@DiegoPino yes, sorry, it shows `strawberryfield_flavor_datasource` as the only bundle.

**Review by DiegoPino — APPROVED** [review]
@aksm thanks. Can you please (Monday AM) make the same pull against deployment-live? Thanks!!

**Pull request merged**